<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IR_TestMain</class>
 <widget class="QWidget" name="IR_TestMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>933</width>
    <height>501</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QToolButton" name="toolButton_Menu">
          <property name="text">
           <string>菜单</string>
          </property>
          <property name="icon">
           <iconset resource="IR_Analysis.qrc">
            <normaloff>:/resource/image/more.png</normaloff>:/resource/image/more.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>Microsoft YaHei</family>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>绝缘电阻测量操作系统</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QToolButton" name="toolButton_2">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="IR_Analysis.qrc">
            <normaloff>:/resource/image/blue/help.png</normaloff>:/resource/image/blue/help.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButton_Setting">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="IR_Analysis.qrc">
            <normaloff>:/resource/image/blue/set.png</normaloff>:/resource/image/blue/set.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButton_UserIcon">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="IR_Analysis.qrc">
            <normaloff>:/resource/image/admin_user.png</normaloff>:/resource/image/admin_user.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_UserName">
          <property name="text">
           <string>Admin</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_Logout">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>32</height>
           </size>
          </property>
          <property name="text">
           <string>退出登录</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <widget class="IR_TestOperatorWgt" name="page_Main"/>
     <widget class="IR_DataBrowerWgt" name="page_DataBrower"/>
     <widget class="IR_DataManagerWgt" name="page_DataManger"/>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>IR_DataManagerWgt</class>
   <extends>QWidget</extends>
   <header>IR_DataManagerWgt.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>IR_TestOperatorWgt</class>
   <extends>QWidget</extends>
   <header>IR_TestOperatorWgt.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>IR_DataBrowerWgt</class>
   <extends>QWidget</extends>
   <header>IR_DataBrowerWgt.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="IR_Analysis.qrc"/>
 </resources>
 <connections/>
</ui>
